src/middlewared/middlewared/common/smart/smartctl.py

```diff
@@ -47,13 +47,6 @@
         port = units.get(channel_no, -1)
         return [f"/dev/{driver}{controller_id}", "-d", f"3ware,{port}"]
 
-    # LSI MegaRAID 6Gb/s and 12Gb/s SAS+SATA RAID controller (only supports passed-through disks on CAM bus 1)
```

Comment:
I wonder where did this originally came from?

There are several controllers supporting mrsas, I wonder if there was a particular one that really needed cam bus 1?

Reply:
Originally old ix-smart had to remove all possibly non-functioning disks from smart config. Now, with rewritten ix-smart one disk that does not work won't lead to smart service not starting anymore, so we don't have to do these checks. Mav told me about `bus 1` thing when we were working on https://redmine.ixsystems.com/issues/35998 but turns out it is not always true.